bats/fb-verify-packages.bats

```diff
@@ -6,15 +6,34 @@ set -o pipefail
 load os_helper
 load foreman_helper
 
-@test "verify no old SCL packages" {
+@test "verify no rh-ruby24, tfm-ror51 or tfm-ror52 SCL packages" {
   tSetOSVersion
+  if ! tIsRedHatCompatible; then
+    skip "SCL package checks only applicable on EL systems"
```

Comment:
Should this also check if it's EL7? It's not really relevant on EL8.

Reply:
We didn't have that before, but since I am re-factoring I can make that additional update.